Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -963,8 +988,82 @@ def get_mapping_fields_command(service):
     demisto.results(types_map)
 
 
+def get_last_update_in_splunk_time(last_update):
+    """ Transforms the time to the corresponding time on the Splunk server
+
+    Args:
+        last_update: The time to be transformed
+
+    Returns: The corresponding time on the Splunk server
+
+    """
+    last_update_utc_datetime = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
+    dem_params = demisto.params()
+    splunk_timezone = dem_params.get('timezone')
+    if not splunk_timezone:
+        raise Exception('Cannot mirror incidents when timezone is not configured. Please enter the '
+                        'timezone of the Splunk server being used in the integration configuration.')
+    return date_to_timestamp(last_update_utc_datetime + timedelta(minutes=int(splunk_timezone))) / 1000
```

Comment:
why not use `datetime.timestamp` or `datetime.total_seconds()` instead?

Reply:
Both are not supported in `Python 2`.